src/main/resources/org/cirdles/topsoil/chart/topsoil_js/Wetherill.js

```diff
@@ -0,0 +1,45 @@
+/* 
+* Copyright 2015 CIRDLES.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+(function() {
+    "use strict";
+    
+    var LAMBDA_235 = 9.8485e-10;
+    var LAMBDA_238 = 1.55125e-10;
```

Comment:
Not that you should know, but these constants belong in their own file. We'll have another curve before long that'll use them as well.